api/intro.rst

```diff
@@ -22,6 +22,8 @@ endpoints listed here.
    * `.NET Client <https://github.com/Asesjix/Zammad-Client>`_ *(Third-Party)*
    * `Android API-Client <https://github.com/KirkBushman/zammad-android>`_ 
      *(Third-Party)*
+   * `Golang Client <https://github.com/AlessandroSechi/zammad-go>`_ 
```

Comment:
Wouldn't be "Go client" a better wording?
At leas thtat's what I guess from the Readme of the repo in question.

If not just let me know. :)